test/e2e/framework/specs.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"github.com/onsi/ginkgo"
+)
+
+type spec struct {
```

Comment:
I would prefer a more descriptive name for this struct, `spec` is an incredibly overused term and its unclear what type of spec this is

Reply:
`spec` is the term used by Ginkgo. But I agree, that context should be more explicit.

How about `ginkgoSpec`?